src/modules/move.js

```diff
@@ -104,7 +104,16 @@ module.exports = bot => {
 
     bot.editChannel(thread.channel_id, {
       parentID: targetCategory.id
-    }).then(() => syncThreadChannel(threadChannel, targetCategory));
+    }).then(() => { 
+		  syncThreadChannel(threadChannel, targetCategory)
+	  }).then(() => {
+		  bot.createMessage(threadChannel.id, {
+			  content: `<@&${config.inboxAdminRoleId}>, a thread has been moved.`,
```

Comment:
```suggestion
			  content: `<@&${config.inboxAdminRoleId}>, this thread has been moved.`,
```

Reply:
the then chaining was done so the message would send after the permission changes were done to the channel, and to be in line with the original code which used the thenable syntax. wouldnt much matter either way though